src/annotator/integrations/test/vitalsource-test.js

```diff
@@ -246,6 +246,28 @@ describe('annotator/integrations/vitalsource', () => {
         throw new Error('Unknown book');
       }
     }
+
+    async getTOC() {
+      if (this._format === 'epub') {
+        const toc = [
+          {
+            cfi: '/1[foo]!/4/10',
```

Comment:
The "/1" is technically wrong, as elements in an EPUB's table of contents have paths /2, /4, /6 ... The distinction does not matter for these tests.